wpilibc/src/main/native/cpp/frc2/command/RamseteCommand.cpp

```diff
@@ -67,8 +67,10 @@ void RamseteCommand::Initialize() {
                          initialState.velocity * initialState.curvature});
   m_timer.Reset();
   m_timer.Start();
-  m_leftController->Reset();
-  m_rightController->Reset();
+  if (m_leftController.get() != nullptr) {
+    m_leftController->Reset();
+    m_rightController->Reset();
```

Comment:
I would check both variables individually.  That way any static analysis tool etc will understand too.